packit/api.py

```diff
@@ -2343,11 +2347,27 @@ def run_osh_build(
         if csmock_args:
             cmd.append("--csmock-args=" + shlex.quote(csmock_args))
 
-        cmd.append("--config=" + str(chroot))
+        osh_options = self.package_config.osh_options
+
+        if osh_options is not None:
+            analyzer = osh_options.get("analyzer")
+            config = osh_options.get("config")
+            profile = osh_options.get("profile")
+
+        if analyzer:
+            cmd.append("--analyzer=" + shlex.quote(analyzer))
+        if profile:
+            cmd.append("--profile=" + shlex.quote(profile))
+
+        oshConfig = str(chroot) if config is None else shlex.quote(config)
```

Comment:
This gets a bit more complex:
```suggestion
        osh_config = str(chroot)
        if config := osh_options.get("config"):
            osh_config = shlex.quote(config)
```